$NetBSD: patch-Makefile,v 1.1 2019/09/11 16:26:17 micha Exp $

Create ${PREFIX}/bin for installation.
Do not force user root, to allow installation with non-root account.

--- Makefile.orig	2018-08-27 08:42:00.000000000 +0000
+++ Makefile
@@ -141,18 +141,19 @@ stripped: all
 INSTALL_DIR=$(DESTDIR)$(PREFIX)/share/eureka
 
 install: stripped
-	install -o root -m 755 $(PROGRAM) $(DESTDIR)$(PREFIX)/bin/
+	install -d $(DESTDIR)$(PREFIX)/bin
+	install -m 755 $(PROGRAM) $(DESTDIR)$(PREFIX)/bin
 	install -d $(INSTALL_DIR)/games
 	install -d $(INSTALL_DIR)/common
 	install -d $(INSTALL_DIR)/ports
 	rm -f $(INSTALL_DIR)/games/freedoom.ugh
-	install -o root -m 644 bindings.cfg $(INSTALL_DIR)/bindings.cfg
-	install -o root -m 644 defaults.cfg $(INSTALL_DIR)/defaults.cfg
-	install -o root -m 644 operations.cfg $(INSTALL_DIR)/operations.cfg
-	install -o root -m 644 misc/about_logo.png $(INSTALL_DIR)/about_logo.png
-	install -o root -m 644 games/*.* $(INSTALL_DIR)/games
-	install -o root -m 644 common/*.* $(INSTALL_DIR)/common
-	install -o root -m 644 ports/*.* $(INSTALL_DIR)/ports
+	install -m 644 bindings.cfg $(INSTALL_DIR)/bindings.cfg
+	install -m 644 defaults.cfg $(INSTALL_DIR)/defaults.cfg
+	install -m 644 operations.cfg $(INSTALL_DIR)/operations.cfg
+	install -m 644 misc/about_logo.png $(INSTALL_DIR)/about_logo.png
+	install -m 644 games/*.* $(INSTALL_DIR)/games
+	install -m 644 common/*.* $(INSTALL_DIR)/common
+	install -m 644 ports/*.* $(INSTALL_DIR)/ports
 
 full-install: install
 	xdg-desktop-menu  install --novendor misc/eureka.desktop
